src/test/java/io/vertx/core/net/KeyStoreHelperTest.java

```diff
@@ -70,6 +68,23 @@ public void testKeyStoreHelperSupportsECPrivateKeys() throws Exception {
     assertKeyType(helper.store(), ECPrivateKey.class);
   }
 
+  /**
+   * Verifies that the key store helper can read a DER encoded EC private key
+   * from a PEM file.
+   *
+   * @throws Exception if the key cannot be read.
+   */
+  @Test
+  public void testKeyStoreHelperSupportsPKCS1ECPrivateKey() throws Exception {
+
+    Assume.assumeTrue("ECC is not supported by VM's security providers", isECCSupportedByVM());
+    PemKeyCertOptions options = new PemKeyCertOptions()
+            .addKeyPath("target/test-classes/tls/server-key-ec-pkcs1.pem")
```

Comment:
Shouldn't this refer to the class path location instead? I assume that resources are loaded through vert.x `FileSystem` right? In that case: `tls/server-key-ec-pkcs1.pem` should suffice.

Reply:
yes it should be like that